lib/callableFunction.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+var _ = require('lodash');
+var request = require('request');
+
+var _isDatabaseFunc = function(eventTrigger) {
```

Comment:
It's strange that you're defining a class (something you intend to instantiate via `new` but then not giving it any methods. I'd consider refactoring this a bit so that some of these helper methods are either class or instance methods of CallableFunction.